code/__HELPERS/mobs.dm

```diff
@@ -300,6 +300,8 @@ Proc for attack log creation, because really why not
 	if (progress)
 		progbar = new(user, delay, target)
 
+	delay *= user.do_after_speed

```

Comment:
do_after() is very often used in conjunction with fixed-time effects such as beams and sounds, and you really don't want to go hunting for them. i highly recommend you do not do this

Reply:
This uses the same idea behind how moodlets adjust do_after speed, and does so to an even lesser degree.